rubygem-smart_proxy_dynflow/rubygem-smart_proxy_dynflow.spec

```diff
@@ -0,0 +1,88 @@
+%global gem_name smart_proxy_dynflow
+
+%global foreman_proxy_dir /usr/share/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+%global dynflow_config_dir %{_sysconfdir}/dynflow
+
+Summary: Dynflow runtime for Foreman smart proxy
+Name: rubygem-%{gem_name}
+Version: 0.0.3
+Release: 1%{?dist}
+Group: Applications/System
+License: GPLv3
+URL: https://github.com/theforeman/smart_proxy_dynflow
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: ruby(rubygems)
+Requires: foreman-proxy >= 1.9.0
+Requires: rubygem(dynflow) >= 0.8.4
+Requires: rubygem(dynflow) < 0.9.0
+Requires: rubygem(sqlite3)
+Requires: rubygem(sequel)
+
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+BuildRequires: ruby(abi)
+%else
+Requires: ruby(release)
+BuildRequires: ruby(release)
+%endif
+BuildRequires: rubygems-devel
+BuildArch: noarch
+
+Provides: rubygem(%{gem_name}) = %{version}
+Provides: foreman-proxy-plugin-dynflow
+
+%description
+Dynflow runtime for Foreman smart proxy
+
+%package doc
+BuildArch:  noarch
+Requires:   %{name} = %{version}-%{release}
+Summary:    Documentation for rubygem-%{gem_name}
+
+%description doc
+This package contains documentation for rubygem-%{gem_name}.
+
+%prep
+
+%setup -q -c -T
+%gem_install -n %{SOURCE0}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+mkdir -p %{buildroot}%{_sbindir}
+
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{foreman_proxy_bundlerd_dir}
+cp -pa .%{gem_instdir}/bundler.d/dynflow.rb %{buildroot}%{foreman_proxy_bundlerd_dir}
+mkdir -p  %{buildroot}%{foreman_proxy_settingsd_dir}
+cp -pa .%{gem_instdir}/settings.d/dynflow.yml.example %{buildroot}%{foreman_proxy_settingsd_dir}/dynflow.yml
+mkdir -p  %{buildroot}%{dynflow_config_dir}
+mkdir -p %{buildroot}%{_bindir}
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/lib
+%{gem_instdir}/bundler.d
+%{gem_instdir}/settings.d
+%{foreman_proxy_bundlerd_dir}/dynflow.rb
+%config(noreplace) %{_sysconfdir}/foreman-proxy/settings.d/dynflow.yml
```

Comment:
I'd suggest plain %config, then it'll get updated on upgrade *if* it hasn't been changed.